### PR TITLE
fix(InputArea): fix scrollbar style

### PR DESCRIPTION
**Pull request by itaytay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/InputArea/test/InputArea.visual.js [hunk 1]
```diff
@@ -40,17 +40,6 @@ const tests = [
       },
     ],
   },
-  {
```
Comment (itaytay) [inline src/InputArea/test/InputArea.visual.js]: After talking to @shlomitc, this test seem redundant so I removed it

**Comment by itaytay** [comment]
Resolves issue #2994

**Comment by nitayneeman** [comment]
It seems that the scrollbar background isn't aligned with the Input's background for the following cases:

1. When hovering:
Screen Shot 2020-01-08 at 7 54 27

2. When focusing:
Screen Shot 2020-01-08 at 7 54 40

Looks a bit odd, UX is OK with that?

**Comment by itaytay** [comment]
@nitayneeman  - I talked about it with UX. I'll fix it in a different PR

**Pull request merged**